sourcegraph/about
# Resources Page

This closes #5651 and adds a new Resources page.

### Changelog
- Added "All Resources" nav item to the top nav
- Added static data, Filters, Card, and template index for the Resources page
- Added new product tour page
- Added `lemon-100` to our colours config
- Updated Hero component with a `centerContent` prop
- Updated our Badge component to meet all the specs defined in the DLS
- Fixed a bug I noticed where nested markup was invalid for our CastStudyCard component
- Updated our Background component to use our simple gradient variants defined in the DLS

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure all links are ok for each card in the resources page
3. Ensure filters work properly for content types and subjects

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202769406180424
